rpcs3/Emu/Cell/lv2/sys_process.cpp

```diff
@@ -415,6 +415,27 @@ void _sys_process_exit2(ppu_thread& ppu, s32 status, vm::ptr<sys_exit2_param> ar
 		, hdd1 = std::move(hdd1), klic = g_fxo->get<loaded_npdrm_keys>().last_key(), old_config = Emu.GetUsedConfig()]() mutable
 	{
 		sys_process.success("Process finished -> %s", argv[0]);
+
+		Emu.init_mem_containers = [old_size = g_fxo->get<lv2_memory_container>().size, vec = g_fxo->get<id_manager::id_map<lv2_memory_container>>().vec](u32 sdk_suggested_mem) mutable
```

Comment:
Missing std::move?

Reply:
This is before Kill, the emulation threads still need IDM state to remain usable.

Reply:
I forgot a reader lock though.

Reply:
Fixed.